### DISCUSSION TITLE
Try to open vscodium in cmd

### DISCUSSION THREAD

**Discussion by trnkobayashi** [VSCodium/vscodium#1749] [Q&A] [answered]
When I type codium or vscodium, it keeps showing this error:

VSCodium.exe is not recognized as an internal or external command,
operable program or batch file.

I'm using Windows 10.

**Comment by anhkhoakz** [comment, marked as answer]
You should read this [https://github.com/VSCodium/vscodium/blob/master/docs/index.md#how-do-i-open-vscodium-from-the-terminal](https://github.com/VSCodium/vscodium/blob/master/docs/index.md#how-do-i-open-vscodium-from-the-terminal)
